Resources/translations/SonataNewsBundle.fr.xliff

```diff
@@ -427,6 +427,14 @@ Liens de modération rapide :
                 <source>group_classification</source>
                 <target>Classification</target>
             </trans-unit>
+            <trans-unit id="sonata_news_block_recent_comments">
```

Comment:
Don't know the exact reason, but all keys have replaced the dots with underscores.